compiler/noirc_frontend/src/parser/parser.rs

```diff
@@ -700,24 +732,25 @@ fn optional_distinctness() -> impl NoirParser<Distinctness> {
     })
 }
 
-fn maybe_comp_time() -> impl NoirParser<()> {
+fn maybe_comp_time() -> impl NoirParser<bool> {
     keyword(Keyword::CompTime).or_not().validate(|opt, span, emit| {
         if opt.is_some() {
-            emit(ParserError::with_reason(ParserErrorReason::ComptimeDeprecated, span));
+            emit(ParserError::with_reason(
```

Comment:
Are we not going to error anytime `opt` is some so this will always just return `false`? Is this intended logic?

Reply:
We're going to issue a warning every time `comptime` is used still so that users don't try to use this feature thinking it is finished